### DISCUSSION TITLE
Scalar Array

### DISCUSSION THREAD

**Discussion by mvictoras** [Nek5000/nekRS#477] [General]
Hi,

if a case does not have temperature, but has other scalar fields, do the other scalar fields get stored in o_S[1+] ?
In other words, is temperature, guaranteed in o_S[0], or are there cases where other scalar fields might use o_S[0]?

Thank you,
Victor

**Comment by stgeke** [comment]
If there is `temperature` it's the first scalar. In case there is no temperature but only scalars they start still from index-0.
